components/formats-bsd/test/loci/formats/utests/SixteenBitLosslessJPEG2000Test.java

```diff
@@ -66,12 +66,21 @@
   private static final Logger LOGGER =
     LoggerFactory.getLogger(SixteenBitLosslessJPEG2000Test.class);
 
+  private int increment = 1024;
+
+  public SixteenBitLosslessJPEG2000Test() {
+  }
```

Comment:
(The pattern's perhaps a bit like `doChange` or `newUserInGroup` in `AbstractServerTest` where we chain the methods to add arguments to cover for our lack of default values in Java, yet another thing the language could have learned from Modula-3.)